lib/src/revset.rs

```diff
@@ -1152,7 +1155,7 @@ fn parse_function_expression(
         state.with_alias_expanding(id, &locals, primary_span, |state| {
             parse_program(defn, state)
         })
-    } else if let Some(func) = BUILTIN_FUNCTION_MAP.get(name) {
+    } else if let Some(func) = state.function_map.get(name) {
```

Comment:
Do we need to make a similar change to `all_function_names` (to use state.function_map instead of BUILTIN_FUNCTION_MAP)?

Reply:
Yes, good catch.